mesonbuild/msetup.py

```diff
@@ -187,6 +187,44 @@ def generate(self, capture: bool = False, vslite_ctx: T.Optional[dict] = None) -
         with mesonlib.BuildDirLock(self.build_dir):
             return self._generate(env, capture, vslite_ctx)
 
+    def check_unused_options(self, coredata: 'coredata.CoreData', cmd_line_options: T.Any, all_subprojects: T.Any) -> None:
```

Comment:
I assume that cmd_line_options is the same type as elsewhere in Meson and should be using that annotation?